packages/batch-submitter/src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -24,13 +17,7 @@ import {
   AppendSequencerBatchParams,
 } from '../transaction-chain-contract'
 
-import {
-  L2Block,
-  BatchElement,
-  Batch,
-  QueueOrigin,
-  queueOriginPlainText,
-} from '..'
+import { L2Block, BatchElement, Batch, QueueOrigin } from '..'
```

Comment:
These types could all go in `core-utils`, not opinionated on if its as part of this PR or not

Reply:
Agree they should definitely go into `core-utils`. That said there's a bunch of type & utility cleanup that we could do and so I'm wary of doing a big refactor in the same PR that we're adding this new feature.

Reply:
^happy to help with the utility cleanup!

Reply:
🙏🙏🙏